robot_smach_states/src/robot_smach_states/rise.py

```diff
@@ -2,69 +2,67 @@
 # ROS
 
 import smach
+import rospy
+
+import PyKDL as kdl
 
 from robot_skills.util.kdl_conversions import VectorStamped
 
 
-class HeroHmiPose(smach.State):
+class RiseForHMI(smach.State):
     """
-    State to ensure that hero's arm is out of the way so that the robot
-    may move his torso up.
+    State to pose the robot for conversations
     :param robot: Robot to execute state with
     """
-    def __init__(self, robot, rotation=1.57, rotational_speed=1):
+    def __init__(self, robot):
         smach.State.__init__(self, outcomes=['succeeded', 'failed'])
         self._robot = robot
-        self._rotation = rotation
-        self._rot_speed = rotational_speed
 
     def execute(self, _):
-        arm = self._robot.get_arm()
-
-        # Get position to look at after turning. Transform the position to map frame since turning will move base_link
+        # Get position to look at. Transform the position to map frame since taking the hmi pose may move base link
         goal = VectorStamped(1.0, 0.0, 1.6, frame_id="/" + self._robot.robot_name + "/base_link")
         tf_goal = goal.projectToFrame('/map', self._robot.tf_listener)
 
-        rotation_duration = self._rotation / self._rot_speed
-        if arm.has_joint_goal('arm_out_of_way'):
-            arm.send_joint_goal('arm_out_of_way', 0.0)
-            self._robot.base.force_drive(0, 0, self._rot_speed, rotation_duration)
+        self._robot.move_to_hmi_pose()
+
         self._robot.head.look_at_point(tf_goal)
-        arm.wait_for_motion_done()
         self._robot.head.wait_for_motion_done()
         return "succeeded"
 
-class HmiPose(smach.State):
+
+class RiseForInspect(smach.State):
     """
-    State to puts the robot in a position for hmi
+    State to ensure that the robot is in a proper position to inspect. This means its head is at the correct height
+    and his vision is unobstructed. (no arm in front of its face)
     :param robot: Robot to execute state with
+    :param entity: entity which is to be inspected
+    :param volume: volume of the entity which is to be inspected
     """
-    def __init__(self, robot):
-        smach.State.__init__(self, outcomes=['succeeded', 'failed'])
+    def __init__(self, robot, entity, volume=None):
+        smach.State.__init__(self, outcomes=['done', 'failed'])
```

Comment:
Why change this from 'succeeded' to 'done'? 'Done' might also imply that it has failed so I actually prefer 'succeeded'.